environments/production/elasticsearch.yml

```diff
@@ -3,5 +3,11 @@ settings:
     number_of_shards: 96
   xforms:
     number_of_shards: 64
+  hqdomains:
+    number_of_shards: 1
+    number_of_replicas: 1
+  hqgroups:
+    number_of_shards: 1
+    number_of_replicas: 1
```

Comment:
Not blocking but curious: what was the number of replicas before?

Reply:
It was set to default value which was also 1 for these indexes.